cmd/openshift-install/agent/waitfor.go

```diff
@@ -93,25 +86,24 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 			ctx := context.Background()
 			cluster, err := agentpkg.NewCluster(ctx, assetDir)
 			if err != nil {
-				logrus.Exit(exitCodeBootstrapFailed)
+				logrus.Exit(command.ExitCodeBootstrapFailed)
```

Comment:
On a second though/alternative proposal, instead of modifying the rest of the code to unify agent's `newWaitForInstallCompleteCmd()`, what about using directly installer's `newWaitForInstallCompleteCmd()`?

Reply:
Intriguing idea. There is a slight difference with agent wait-for install-complete. Inside it, it [calls agentpkg.WaitForBootstrapComplete](https://github.com/openshift/installer/pull/7678/files#diff-d5f661b569db890ae4aee3bef9355be7894369a4e72c480e8d6b7c0f968bcb2cR92). In addition to kubeapi, agent bootstrap-complete also looks queries the assisted-service rest-api for additional status information like printing out the host validations. I don't think we want to lose this additional information by switching to installer's `newWaitForInstallCompleteCmd()`.

Reply:
Considering Zane's input, I don't think we can remove the bootstrap-complete from the agent's install-complete because it will break existing user's expectations. Therefore we cannot use the installer's newWaitForInstallCompleteCmd() directly.